metadata.yaml

```diff
@@ -38,6 +38,9 @@ resources:
     type: oci-image
     description: OCI image for Jenkins
 requires:
-  agent:
+  slave:
     interface: jenkins-slave
```

Comment:
Is it required to support this relation? We're trying to remove references to non-inclusive words in our charms, so it would be good if we could avoid this if possible (presumably as long as the jenkins-agent and jenkins-k8s charms talk the same relation that would be okay?)

Reply:
@jdkandersson shall I drop support for the old relation since there isn't a strict backwards compatible requirement?

Otherwise, I could suggest that I just rename this relation (keeping the interface) to something like `agent-deprecated` so that the old interface could be kept. Thank you for the review!

Reply:
We should maintain support for the previous relation until we have done the production migrations to make them easier (one side of the relation can be upgraded and then the other rather than needing to do both at the same time)